source/grid/grid_tools.cc

```diff
@@ -1362,7 +1424,7 @@ namespace GridTools
     // all adjacent cells
     std::vector<active_cell_iterator> adjacent_cells_tmp
       = find_cells_adjacent_to_vertex(mesh,
-                                      find_closest_vertex(mesh, p, marked_vertices));
+                                      find_closest_vertex(mapping, mesh, p, marked_vertices));
```

Comment:
nice improvement. is this worth a changelog entry?

Reply:
added a changelog.